### ISSUE TITLE
Server Crash when kicking player on join

### ISSUE THREAD

**Issue by Cosmic5173** [pmmp/PocketMine-MP#4594]
### Issue description

- Expected result: Player is kicked from the server, server resumes normal activities.
- Actual result: Player is kicked from the server, the server crashes.

### Steps to reproduce the issue
Kick a player during the ``PlayerJoinEvent``

### OS and versions
* PocketMine-MP: 4.0.0-BETA13+dev  (git hash: c72d66f3702595f6f9a224b36f5dd00a9a8477f4)
* PHP: 8.0.12
* Using JIT: yes and no (happens with JIT enabled or disabled)
* Server OS: linux
* Game version: Win10/

### Plugins
n/a - the only plugins installed are DevTools and the plugin that is doing the kick.

### Crashdump, backtrace or other files
Code:
```php
final class Main extends PluginBase {

    protected function onEnable(): void {
        $this->getServer()->getPluginManager()->registerEvents(new class() implements Listener {
            public function onJoin(PlayerJoinEvent $event): void {
                $event->getPlayer()->kick("LOL");
            }
        }, $this);
    }
}
```
```
PocketMine-MP Crash Dump Thu Nov 25 01:47:27 UTC 2021

Error: pocketmine\network\mcpe\handler\InGamePacketHandler::__construct(): Argument #3 ($inventoryManager) must be of type pocketmine\network\mcpe\InventoryManager, null given, called in phar:///home/container/PocketMine-MP.phar/src/network/mcpe/NetworkSession.php on line 713
File: pmsrc/src/network/mcpe/handler/InGamePacketHandler
Line: 139
Type: TypeError

Code:
[130] 	protected $lastRightClickTime = 0.0;
[131] 	/** @var UseItemTransactionData|null */
[132] 	protected $lastRightClickData = null;
[133] 
[134] 	/** @var bool */
[135] 	public $forceMoveSync = false;
[136] 
[137] 	private InventoryManager $inventoryManager;
[138] 
[139] 	public function __construct(Player $player, NetworkSession $session, InventoryManager $inventoryManager){
[140] 		$this->player = $player;
[141] 		$this->session = $session;
[142] 		$this->inventoryManager = $inventoryManager;
[143] 	}
[144] 
[145] 	public function handleText(TextPacket $packet) : bool{
[146] 		if($packet->type === TextPacket::TYPE_CHAT){
[147] 			return $this->player->chat($packet->message);
[148] 		}
[149] 

Backtrace:
#0 pmsrc/src/network/mcpe/NetworkSession(713): pocketmine\network\mcpe\handler\InGamePacketHandler->__construct(object pocketmine\player\Player#24646, object pocketmine\network\mcpe\NetworkSession#24211, NULL )
#1 pmsrc/src/network/mcpe/NetworkSession(704): pocketmine\network\mcpe\NetworkSession->onClientSpawnResponse()
#2 pmsrc/src/network/mcpe/handler/SpawnResponsePacketHandler(44): pocketmine\network\mcpe\NetworkSession->pocketmine\network\mcpe\{closure}()
#3 pmsrc/vendor/pocketmine/bedrock-protocol/src/SetLocalPlayerAsInitializedPacket(53): pocketmine\network\mcpe\handler\SpawnResponsePacketHandler->handleSetLocalPlayerAsInitialized(object pocketmine\network\mcpe\protocol\SetLocalPlayerAsInitializedPacket#88641)
#4 pmsrc/src/network/mcpe/NetworkSession(395): pocketmine\network\mcpe\protocol\SetLocalPlayerAsInitializedPacket->handle(object pocketmine\network\mcpe\handler\SpawnResponsePacketHandler#93952)
#5 pmsrc/src/network/mcpe/NetworkSession(358): pocketmine\network\mcpe\NetworkSession->handleDataPacket(object pocketmine\network\mcpe\protocol\SetLocalPlayerAsInitializedPacket#88641, string[2] q.)
#6 pmsrc/src/network/mcpe/raklib/RakLibInterface(192): pocketmine\network\mcpe\NetworkSession->handleEncoded(string[5] c*d..)
#7 pmsrc/vendor/pocketmine/raklib-ipc/src/RakLibToUserThreadMessageReceiver(42): pocketmine\network\mcpe\raklib\RakLibInterface->onPacketReceive(integer 0, string[14] ....8.jM?'....)
#8 pmsrc/src/network/mcpe/raklib/RakLibInterface(122): raklib\server\ipc\RakLibToUserThreadMessageReceiver->handle(object pocketmine\network\mcpe\raklib\RakLibInterface#23056)
#9 pmsrc/vendor/pocketmine/snooze/src/SleeperHandler(123): pocketmine\network\mcpe\raklib\RakLibInterface->pocketmine\network\mcpe\raklib\{closure}()
#10 pmsrc/vendor/pocketmine/snooze/src/SleeperHandler(82): pocketmine\snooze\SleeperHandler->processNotifications()
#11 pmsrc/src/Server(1615): pocketmine\snooze\SleeperHandler->sleepUntil(double 1637804847.229)
#12 pmsrc/src/Server(1028): pocketmine\Server->tickProcessor()
#13 pmsrc/src/PocketMine(301): pocketmine\Server->__construct(object BaseClassLoader#2, object pocketmine\utils\MainLogger#3, string[16] /home/container/, string[24] /home/container/plugins/)
#14 pmsrc/src/PocketMine(324): pocketmine\server()
#15 pmsrc(11): require(string[60] phar:///home/container/PocketMine-MP.phar/src/PocketMine.php)

PocketMine-MP version: 4.0.0-BETA13+dev [Protocol 471]
Git commit: c72d66f3702595f6f9a224b36f5dd00a9a8477f4
uname -a: Linux 54af742a-ecc5-47cb-a44c-9cb0de7c0c8d 5.4.0-29-generic #33-Ubuntu SMP Wed Apr 29 14:32:27 UTC 2020 x86_64
PHP Version: 8.0.12
Zend version: 4.0.12
OS: Linux, linux
Composer libraries: 
- adhocore/json-comment 1.1.2@fc2f76979f0a44a5f5bc2a2b600d0762fe0e78e7
- brick/math 0.9.3@ca57d18f028f84f777b2168cd1911b0dee2343ae
- daverandom/callback-validator unknown@unknown
- fgrosse/phpasn1 v2.3.0@20299033c35f4300eb656e7e8e88cf52d1d6694e
- netresearch/jsonmapper v4.0.0@8bbc021a8edb2e4a7ea2f8ad4fa9ec9dce2fcb8d
- pocketmine/bedrock-data 1.4.0+bedrock-1.17.40@f29b7be8fa3046d2ee4c6421485b97b3f5b07774
- pocketmine/bedrock-protocol 5.1.2+bedrock-1.17.40@5f600ce446f357391968c8ce92e3be989aa1dc9e
- pocketmine/binaryutils 0.2.2@f883e1cf9099ed6a757a10a2f75b3333eeb2cdf9
- pocketmine/callback-validator 1.0.3@64787469766bcaa7e5885242e85c23c25e8c55a2
- pocketmine/classloader 0.2.0@49ea303993efdfb39cd302e2156d50aa78209e78
- pocketmine/color 0.2.0@09be6ea6d76f2e33d6813c39d29c22c46c17e1d2
- pocketmine/errorhandler 0.3.0@ec742b209e8056bbe855069c4eff94c9734ea19b
- pocketmine/locale-data 2.0.16@8bd74825ac685446f67ea20999ce756a2acd5a30
- pocketmine/log 0.4.0@e6c912c0f9055c81d23108ec2d179b96f404c043
- pocketmine/log-pthreads 0.4.0@61f709e8cf36bcc24e4efe02acded680a1ce23cd
- pocketmine/math 0.4.0@6d64e2555bd2e95ed024574f75d1cefc135c89fc
- pocketmine/nbt 0.3.0@98c4a04b55a915e18f83d3b0c9beb24a71abcd31
- pocketmine/pocketmine-mp dev-master@c72d66f3702595f6f9a224b36f5dd00a9a8477f4
- pocketmine/raklib 0.14.2@e3a861187470e1facc6625040128f447ebbcbaec
- pocketmine/raklib-ipc 0.1.1@922a6444b0c6c7daaa5aa5a832107e1ec4738aed
- pocketmine/snooze 0.3.1@0ac8fc2a781c419a1f64ebca4d5835028f59e29b
- ramsey/collection 1.2.2@cccc74ee5e328031b15640b51056ee8d3bb66c0a
- ramsey/uuid 4.2.3@fc9bb7fb5388691fd7373cd44dcb4d63bbcf24df
- rhumsaa/uuid unknown@unknown
- symfony/polyfill-ctype v1.23.0@46cd95797e9df938fdd2b03693b5fca5e64b01ce
- symfony/polyfill-php80 v1.23.1@1100343ed1a92e3a38f9ae122fc0eb21602547be
- symfony/polyfill-php81 v1.23.0@e66119f3de95efc359483f810c4c3e6436279436
- webmozart/assert 1.10.0@6964c76c7804814a842473e0c8fd15bab0f18e25
- webmozart/path-util 2.3.0@d939f7edc24c9a1bb9c0dee5cb05d8e859490725

Loaded plugins:
DevTools 1.15.0+dev by PocketMine Team for API(s) 4.0.0
MultiWorld 1.7.0-beta2 by CzechPMDevs, VixikCZ, fuyutsuki, kaliiks for API(s) 4.0.0
TestKick 1.0.0 by  for API(s) 4.0.0

----------------------REPORT THE DATA BELOW THIS LINE-----------------------

===BEGIN CRASH DUMP===
eNrtW2lz20aa/itdUqoiZwgK98EpZe1IzkQ1cqy15HV2Ri5Xo9EgEYEAg0OHM/nv+7zdDRKkZMfO
ZKvmg12pmGh0v/fd8K97ed0seffuRjZtUVd7M3+y1xVLuTdzQi+KbT/2o6nrRY7nTfb6lX5FK25g
h7YXRkHsu3hV8rZ73jR1szf7da+7X2HX3iX+0muTvaVsWz6n1VUtrmW3LCp5dVXJ7rZurq+ulmKF
xwWvslI2V1en1d/4Up5z2vmDXpzN3r0TddV2TS+6gycz9qyZ90tZdWzfYwdfFdUNftfN/QteAU/z
hC37tmOpZHXOiB72EbynO4cnrOrLks0LLE+Y4GUpM1ZUbLXgzezw8HBRL+UhiOk4oDWH5wryC/y2
XpxPadNh24hDg+SQcBz+qB8uIAaIGZtWrK5YiTMMsoWAcmD8viiVhP44GiPBw0cESDgJj8GxpLOf
cx5niVwYhpfARhou8Pufe/s2+wCsbZYPwCa09ge1b3071n6d/ixFtwVrVfJ7Onuu/t53/dAPJ+yR
jTtIt4nEOddxJuzH12dn7Ak43nc+lTvb/zh32/utb+vquCxgdRcrflu9ku0K3MkDhdNlv6OdrTNb
cjrwP5OMj2z9VZR12zfyN02VZ6iCT2R1c7g5eJjKrMGTtWrqrhZ1qQi/kN1ZDdfRCnnWnlZFV/Cy
eC8zTfFB8Kn28GF2rW/1po8gO/h9GxjoBqbfI3o/jkPfUfLwP9EyvCT4OJ+fgX7g9xOY+gTp7Seg
zVXMBJ/KTBB/nn1pMk54x43W/x/UMWGIC0U1/6f7lv0yVfyEH+Kn4ddlkR6+4tdnRXpadbLJEcgO
nMT9I3w9r0SdwcYM/uAtE99kU01C9EF/0TRYxUrTp2m5rF+3srlcNJJnL3SyfCWFRAqCU/8OcRrg
1dUOVxRktJwMpIMCr5DfmL0WmeO/ZVP8iac/v/ivr+mXIj7+XPm5ROJABxi5IdMDhwNRH2HvM4z6
A4zuu54dhIrw5INSb6u6fi91bCqlXMlmiJmO6/1B+X7CmZ0o6th/gMB4V/96Kxxkax/oQRyGcH+s
uyIvBO9gqq3BO05kF0o/B07oBJ8IuKXn11VXlAdZ3aelZOPy0E00DvcRHLa7GzH0G+vbrhDX55rg
ujFUeiMIm3rnwLOdDwB5pCz4jrfyGPVoe1bzjGqBR+uAHry0V1cvUFud1XM4xb63cYrwLdutvjZB
xn/4clX286JqDzUT/geYcHdzs/YTw7oJwQcOsdrIX/qikUNgCe23n1l8bpap7nuy9/a3yZ78Up//
G/X5n1E3QwnaVN5BHnV5I0lAgFrV2DC8wzN+U2IhXTmejYWrjrIibBiMfkW91qtivuhQQYrrS7Rk
7IjZU/uvOOZ4jtp++M037OkNbxjC7mknl5cNr1ouKCJQMv6XEuA3h+qE+3EEtB8I6ITG4GkKHc/f
RpXW9RpmoGEiVBSCfYUmU8gX9Y28uK8EYOW8bKUGFg7AIkNFccM7yXZVzh5YkD4eD8eTMcK8rxSr
bGyTuopgX+lWAUX+lsrZV63+MfkE3E9+JZy+1sxV91W3KFpEX43gaEChKPSd7U0GC+0yP/U2d3vb
Lkba/6gEfE8d/E399o00/OBRaWgrvZR33QH9T5spqFV/P2EzpUHNWmjIKfID8x4Bm9z06OiIbQ7P
Zpf/e/783fEPzy6NTCJz8KprZNc3kOtYONa3YsG7DUgTdp5oVmJzVjNDGt377Uur+aXV/NJqfmk1
v7SaX1rNL63ml1bzS6v5H99qDo1MS/3Liby5RFWpflecbo82S5O99bXTnjN1gqn9l0zeYJn33aJu
Wir6NtBRdvIlQeergt74U/Q89CwrDjPJ9mbQjZzsZTCnKqPDeNfWeXcyXlhy1WENNFxdjagpoUW8
u7h89ury9TkWbmXaFh3RvOi6VQuZzItu0adTUS9HfB+uYaBcfdGXXfGmbspsxPJocZvpaGpbqey4
u8318XspFucvAJfo+p/irrg+/oe6Jbrvu7a/LvD7GumpuG7/LIkIQrlaQgEQypLovSV6Ybdj2o2I
zl9eXL55+ers5PeENGLkcAQIcrqUbfd3+OVISuulbRkRU2Pp/EkMD9iurvT/yT9/j8O936iF5w3o
RWRWqpo+4hc4Y1mt7Dp4VDvVxFmQrJX2RZm1R47aUNXWbfGeN5l6yopWbUOfXiiW7jpZkQyU6xzX
DeGPwbBDtpLxbusZCeBuWY5XanDctltLK7ENpP2lBFveeOk9AI2fU7Hk3WK8giIJogSP47VFX12r
MEdtrD31psShaO5XXW0W1DYzeNqc65stArN6iUfXtj3HcRMzdSmqvB5vwhpkv7XSrcaP82zrabn1
csHbLXbajhM3W0d+bpXlbYQrb2SZpYYTW4s7k3mplYBdanGZ6hg6Prqsm04Bo01q4R5Srwid+cXW
ey/Oz7a1VXVb0lEHivHK+cnLrRNZ/U5t2l3Uit46udjW4Kpui7stuSijbrf2dKqmojXfOGUjoUIy
0nmvZTjI4hWkI9XUYQvoYllvaeOiWK5K+dOLLcaLeVpUvLnHGuxmGmKpq69lhWJ8i+Ydg8cjkfdg
z21T7NjLPVcnXQAnO31frHQCSqY+Hv8B82YvzwUXi5HIyO8XK2OLZFYSKZGXo+i1FQbIdVAcbL6r
0CKzvnt++UzNE1Uk2JvZYLd9R1nPhK6hJdmb+ZEDNAXNB0XkZmGYe5HtBkmQh3nCXddPvTAPssy2
ecJRHEU5Ud8bas6Kqr9jgc/zyHe5JYUILD8SqcV9X1iJSO1MRsIWccaCKWiz3MRSPBWC7Xue9Trt
q65nFy/O2RuZsWerhrkJc/yZ587ciL2+PGau7drsLg7fhYSYPi8YBxKpjNxfG89i9a5uB8IoPtFD
aR6QKmCBsnkHx2p4U0gV9Hi2gCgaeUgOaWGPGZc65EtPc+HmUZhESW6DJx7kQSpc7qahbWd2FLq5
tGUUy4iEDa7QVehIZk+TqfdU8CDKnDhH0ZjHfh5FUeo6YSwyJ3EcEo50Pd/jUsVaaBFRAumMZs0p
+hzrBskXMZjm6RD5dVXfVk/N3xSQ5k3dtvIQTPO2ogngjUuR4ylEliS25wkvyH3PtmUaBqGMZCzj
WOSBmznQc+ITVhT5jWwlb8RC8b/kq5Uy4xtlSU/jNBW26/BYZqkrfR5J7uYxz/ycJ1IkmZBuLtI4
U879YIAC2rmSJID9ZViEXKOpbz/N3SSNUhnn3LP9MHOl9EXou44fB2kSpR4kbUNg/uOgNxa8F5Ci
HoCHAdu2kL4Piw4iL3ESyD0WMnGll8okTjh3MpHIHfAqJqgko5TokgnEsScdkSd2ksgs5FEQcceG
HKIg9fBHytQVWZ5sQ3pUiVRmeE9DP4ojH0YVhqngkGkQx4HruzIOhOsJN5CxCALu7gCkFqCsTewh
0uynfiIhPS9JPJlneeolIvNsV7pOEGaBDdCxayewzx1IdVlvYNhJKkPJwywKc8jGy8LYgekkmZsI
1xV+KJxIOtkONeqax8xkhpT1VAqEgZRQxuhfUyg3COwwEb7M88QXSeT5kjtJug2qpJGIHIzFJV8O
YXhZ5MduwEUYB6TEkEwPGkiEjIKQu1xkAXjfBTVXxPhETCgSxxU29BYEIgYHnmPHUsD8oyRNwty3
fWH73gMI1igFaVChk0fElMg9aEy4vgRH0iYaYBCxzR34gSd2vGAdCRSILPSlGwRBCktPApnZrh9E
CAlBhsO5cDzQmORiG0SVdmvhJrHwue2nsIzECSSiSuxlXmoL6C914ZoOT6F+ZxvC5qelShSqD5e8
Ra56+hnx/sGwRydhH94hPR6HjgODjmzp5FyIMHQD27cdxDzfjyQiSMqleAwKjYxMOneeJq7LQ9/3
wVEoooxzHtB/sec6NgxQCj/yYi53hKwb/KEefGpzESNkw/Ad4TsJd3KIHU7mZ0HsBRSGg0S6ygJR
XrfynpxhU0M4yuEF/kS+lIH03BhVYgp38u00cGDUUsYQehqGwuYbIH1f6DyEAIyUkaRplKeBF8dh
4uRZ5EUwDt/PROpnIU6L3PWznI4v+mXL+XD+YYxv75d5Xd1DjeU9StLSGqrbG5BKVgH3zJIgSiKZ
IAR5cZ5lbmp7YeKlQY70I0MIFBb2GCwkjthew3KeOg5qYt+TmcMpSnJAS7h0XIR45BAkLhiKj5D9
IVjOiC4ZwiiS3MvI2HNkosSPvTx2bOELD0R5oRsl+L9ufJY1OpTuEAFONib3UvYJk9AXEayBBjOO
D3t0YQQS9USeOUHKUzt3YnjVFpAV3M6iCK6CCdGSJV6SRzKD4wqYRJomglJvgPIkyJAUQVtiRwCj
+q61bU3vVQG3z6hnY6Ku8mLeN2oIxahlYHndsK167KraZ5cL5FM2tGWMN5J1C8nqSrb4wTsmeFXV
6lK6qETZZ/pWWY9OpkhryL8dShOCdVEv9dX1Q5gtz+WE1XjTtASSpQhZ1+y+7hsDixU5W9ZZkRcK
A0oc1NFdeU+Af5S3a3CH6DE4muaW3dbV1x2jCoA3DJ1wjQhWUB67Jwrphk3zfbuQFetX84ZnADC9
qq6qAdjsqmJs/81CEmGsq4EFdS5AmGkTZEB3xNTTZPq1IlWnAAZ5lrJT8kKcKoQ0l4m0siTI6orX
Kjl1AnM501e89KJd9F0Gl7HMXd+MXe3pkRqjeaXMrvYUZc/Ksr5lZdEStcxMkNhNwdl/97K5pz2/
0A/LvBqh2H+uOmxziIEwRmUjg8ogFQKX3jMjS9pvGvL16zGoN3AZmAQEy7OMoQLT8ie50hSRLSWn
Gav6sgFSTyVRDenIX3pektzUVwkMpUUv2YFBilL5CUHXGKXVyBX6Qgtyp3HgDK8V7h/7ZQqxwKye
tfeVuOTtNaNBLwxpqja8hriUbQ+c0rZFU1d1D/VhezthamrDdHvSqftsKq4bc+1MsoFD0tqyr8zg
VgO/UPoFTzCzpr4rYGISYgBLynSrNXEkW0PQaU7GSnvIKCes6L6G+LpGHcskfVCgbGbnMDsADz0c
x1ViUWxYhtOZArVrrFxZR9NXFakzo4a8Xikl6JHKlJ3IG/ObLenbBSYatPqTsfcZyyGJNE2/6vSq
FhgJBs26NHwp5cIrVc+RaY/gN3WRgezHKSAbIZtD4V/AuqYjQ9tMftaWdlUtJdDda6/8W1mnsB6a
VDG9DkdYFh0591LOeXrfQeLsQovaVsLV4yJ1/JK0dlsoySuLgtfeWhrQYGSt4jAvQCJCkqQvCQhM
3kg5oNRGDm315KdsXkNRNfkpWQUhmisyLUXajNnEBbCrGKzrsv94Dmj09zH6F7zJ/j36265eKRJM
/FyTZMC1DOFjRek0WxNEWHepUhHgjXIJQ1z79Z9AnnIKRQ+FCyZ6pJ4KqYfJOyl6FXprTa92R0pV
693yTkiZtRt2dNR6efl8xi5/OL1gZ6cvTi/Zs/Pzs9PnF+z8+Sv25uWrvz9/NWHYxC5fYtdz9uaH
l2fP2fmrl8fPLy6mu/7/qDDOZVPUKhlTDG6JPbGQKhorWazjrAOxoxzIVFhRWyxEQakjrAJ1XFfg
ErEZhkTcfsjOtH3dLiir0pdgiD56m4KsgSDqbmI4jW8MjpeUPijHqwPTh7vJDMzIGkpbSUGlgJYr
kau1qs+ueYBlyVW7YdWzDa/tk8dJMpx7hvM5b1I4hbWpqlXseVy6ysuIHHOKmVMI20te9arwGFOC
oqyHBT6ZDOlg1wQHI9SZNc8BS/O73HjflL3U4V6bOyqHui+zNS0F8sdtxTb0txrwSpEPRkPbNrwC
2/cqTjzIj8p0NIQ1b3lTL4c8q0+bHdbYMMcaJm6MJh/YBnvEpLas42IoFkllDTCRGRpDzvrlSldW
unxj5oVFLwZ9ndAmc0DRrsgcOMATMjxcnT8iYORr5DxguIGRtW0vKSx0C2htPlSaFK8I24aqyaBI
XR6YgnKqqaGtH5TTkt9Z6m6gHWg/5itaLZb9EukVmbUh4B2vUFFCkcx8Q6eo2MiV/MWIUnnMQxtD
ob0yFCmEFlXAPfKtv9bYSa1f7do0lUUU8zR8vXlwIQ3LbBxzpooGS82I17ydgBRVpqByJOmBOl1b
aCj6mzemz2gON9zhmJGnuQYymGnzgFcLsKQWwMD9OKiP2eFVZT4imJkEgapsUZe6ukx5JxaqClck
o2CCCZkUo2Kb7nrMW51YUrmuM3Vm2x9raXhFYbW57wj4hFnOSIFKeQqx1Q3EIGwHoYnamxJW3cZQ
9ZhpzqmPUTU+HcaiIWvKfkAZqL6cpLqOHZ+/nuAsaEjhX7dFBrNXZYKOnmv4loI/Y+FQcW+K8VZJ
Z7R3yp5lCFQLXuaHaCN1h5BJ2LBBBaSMBoNkYCNH3CS9EbCthuaOohpVl7yc0je87PV5dT7U2Zvm
j9oIIuoWSZMO9qtqZZnnncbmhfG6tngv1/qkinxQY0sJz1RLqNi1eYxSq5+4+tCTKTs3Z0rezNUB
brpQYwsKY97wObEAnUBs7apU9Qq1lkv6ZBvOzhvSk/46UffK9DWCbDv24vK1orQFS8R22neMenOl
yg063VdMh1iz7HqLTs0UtePWUFbqNlL5es5oKIBKCLXqwCZ63zwvxJSpLLXgVASwYgnZdKQ7aEP9
Q1mEqYmiBUq7UUQvSBKIuioQI0VzFc7Np726lMDeztgocSL1bMCEOtNInbxUJdLJ6cWz71AdqWqq
r5QNUcDuKVKqq0pdJBRkjOYb8mzcamz43Ph6JtN+Phv6tW+ZQ32aqVYXCFfqPTPteasHCuTNVVvr
5G08wiVgmmoN7XhRo4EnLxzaNDk04iqK09yYtfxmyBb4ia5bfwhM7zaBjQhT91+6aDPzmJ/rotLD
ACWpr9lPaX1Hnt+w0xN28NPr05Mn0DvcHjRg/y3URoopkO7hHNS+NZmRjy4/4A31RncVvP2n784G
+Ka1G618rZDRjZoqHJR6TSmieKuH6ZBJo3egz6IPFY7qPFdo8a7I7627vsg2ClHytLYHM5dqsqJd
Tf+bbC0Dtdf4Vd+aCY9AEOl0mDVnLH1mxsxdNaHRKcSER4OFciCv5uv2pKJj5W4qngy7bgp5a60T
9GPDMC3cZ8u6p+iR61xngglENoiWcjtFRzUFgfrpt0rOOPxKperRYcV4JXXXnUoNLJU5+X5L35gq
V1oMdqZMi5a3k75mnxCt2UcMZHyXVPMlG7BJ5NPHqLQfJ5M3gJQxPpg7iB0O099raryd4U5a1kK5
b1tkOmcQPxw6hcG1fTqWwpySKHbITKdO9bBObWrKZIyDCyFLqToFbd4Ka1lcAwW1yCoSlRysz5v6
tlvoOAYUnC00tFGQg/e3nU7JiliLpDGQZm1Eo1k7PWlHfFGYJs1raEzfnK7ZNNOmocxRZ7aUBHgW
c9k+yESXvFHkZqK11uX0oTJN8LrlheplkUzQ6mkrWtWrvhzcxjxQvld7TNrwVIxTbRBxqTBRxqXI
hZLANBi0YmZ4M1Pq0MhRtdZ6WVcKxrJAyCZCZ5ugR55p0Ve9O0vrwLC9bD4/GOV0xANLf3O13rmA
3maalOlquVxNi1pRjFB3v6TmEK7c0YxVtBvSKU2Z92zzXrGgwhyfzxs5N9NEM3rUExb6txnXu/wp
+hBrXp68pM8LLZMUdH1A4etW6g6Ew8lvFcZWEb5hQK1NRwN/5Om15PsVff2klbONlFagUb3BWBNK
Ifp2QCWzUcI5psG8HM/xJsgOqwWHX0gK7Q0RMRSmSBm5bNB8WBQYK8qGtGtIXTpx8K2poPnyY5Rr
e5TgKG1UaB0imAGnlKzfDxgeKFVzNdZqVyw3CcQkYwJK+8noKV+QEimKE0I17jSHhtH2dIPAvBlj
GOT28XRvcolWpHaA0SDJwAD4zkhzGBktzKKOcM26kTDBgcbv6NUyVQUV+RoQXZL3jb5soWAzTFJ0
4AU8E5yGdIuSjLemTUNlzemTI/NE9Q9oNLqaDUrT72CiNM2bsX+qwgU160SN994SUNWBbXL3+BJI
sUGFdKPiO01RdOwim5Ldo2lUhWfVPbaqYVDdN30vCVpNDTAmX+NVNOifgEhe4Ns6pW4wztj3Z88u
9SJ1GhKKdv9qPsmYBMkdOKrkxLvLiqabqJj7V0cXevqaZTyz0D5MeQ6iNQNF3hurK6Escc/0F9J9
M4TdoVFr1vczKqYADrKRUBdg7SAna7jcgQnoqn7/exVMFCq1fwuIKovNyrvRiqXKy1HdV+vSX94h
tLWmxxXX1CW1qp9DbEsL9Cf3uo5T+4hfkIIOTCcK7SqaSwUfhVGzDjx087qrVLqiPN+o+FjdU6qL
uisULaCsZz/WN8xxmO3OnAT/Dd9VObRhWXfZ0RverCAk+liVCNI3Z/RWY7PI2Y6cxEHmwuLtouik
RZdoR7BigoGuyNRgRw6lL6zNOU0FMnmksOiLu/UaCldapSksXdUMz6ublYGYFdQowTTvjxTKtaWt
89YRLesal0rnI/0dcbOpe2U22kK35kcnz79/9vrsklZNI6Pu+gxKFfXJB83zVlV6FNPSqPJWW4aL
yCOJ5LQtrZtQycsbIStWWNQH9377PzmMGgQ=
===END CRASH DUMP===
```

**Comment by dktapps** [comment]
You're not supposed to kick the player on join anyway, it's a giant waste of resources. Cancel `PlayerLoginEvent`.

**Comment by dktapps** [comment]
Fixed by f7d0d16eb3b1f4be73535a5674b2da8a2b799a94, although you really should be using PlayerLoginEvent for this, not PlayerJoinEvent.
